frontend/src/components/IstioWizards/WizardActions.ts

```diff
@@ -103,13 +104,13 @@ export type WizardAction =
 export type WizardMode = 'create' | 'update';
 
 export const WIZARD_TITLES = {
```

Comment:
Adding the `t` function to an external variable does not translate the variable. This is only used to generate the translation statements in `translation.json` file. For external variables not defined within a React component you have to perform an extra step: adding the `t` function to the `WIZARD_TITLES` variable in the Kiali application.

```
${t(WIZARD_TITLES[this.props.type])}
```

I have added some suggestions above. Look for all the WIZARD_TITLES variables used and add them the `t` function.